loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -645,25 +641,19 @@ void binary_update_test(void)
 void binary_update_aging_test(void)
 {
 	int ret;
-	binary_update_info_t bin_info;
+	int randval;
 
 	while (1) {
-		ret = binary_update_getinfo(APP_NAME, &bin_info);
-		if (ret != OK) {
-			return;
-		}
-
-		/* Copy current binary with a higher version and valid crc. */
-		ret = binary_update_download_binary(&bin_info, DOWNLOAD_VALID_BIN);
-		if (ret != OK) {
-			return;
+		/* Iterate update test with random binary */
+		randval = rand();
+		if (randval % 2 == 0) {
```

Comment:
To make a more easier than now, actually powercut stuff  works random time both of turn-on & turn-off, so I think random is unnecessary here, I mean Test Equipment already uses its own random logic.
Testing can be done sequentially.

Reply:
This is for random scenario, not random timing for update. It makes it to run micom update or wifi update ramdomly,.